services/api/routes/authenticated.js

```diff
@@ -190,7 +192,8 @@ var routes = [
                     Joi.number().integer().required(),
                     Joi.string().regex(/^\$(\d+)\.(.*)$/)
                   ),
-                  title: Joi.string().max(256).required()
+                  title: Joi.string().max(256).required(),
```

Comment:
I actually talked to Ricardo about that being weird. He said we can up it to 140 char limit on descriptions.